lib/desired-caps.js

```diff
@@ -125,6 +125,12 @@ let desiredCapConstraints = _.defaults({
   },
   useJSONSource: {
     isBoolean: true
+  },
+  language: {
```

Comment:
new capabilities should be documented in README

Reply:
does this capabilities need to be added ? they are already in the appium-base-driver

Reply:
I can remove it

Reply:
@mykola-mokhnach I actually don't have to update the README since this is in the [API Reference](http://appium.io/slate/en/master/?javascript#appium-server-capabilities) Maybe somebody should remove the (Sim/Emu-only) part since this is also supported on iOS with instruments and I think it also works with Android